.gitignore

```diff
@@ -18,4 +18,4 @@ yarn-debug.log*
 yarn-error.log*
 yarn.lock
 
-manifests/overlays/dev/
+#manifests/overlays/dev/
```

Comment:
Do you want to keep this as a comment or just remove it?

Reply:
I wanted to wait if anyone objected to the location of the service-monitor.yaml and mock-server.yaml location

Reply:
Yes, you want to remove this line.